### PR TITLE
feat: Web 924 explore use real data for the sparkline price charts

### PR DESCRIPTION
**Pull request by lynnshaoyu**
Screen Shot 2022-09-15 at 1 35 22 PM

### CODE REVIEW

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 1]
```diff
@@ -543,7 +561,20 @@ export default function LoadedRow({
...
+                  getY={(p: PricePoint) => rdScale(p.value)}
+                  curve={curveCardinal.tension(curveTension)}
+                  color={pricePercentChange && pricePercentChange < 0 ? theme.accentFailure : theme.accentSuccess}
+                  strokeWidth={2}
+                  width={width}
+                  height={height}
+                />
+              )}
+            </ParentSize>
           </SparkLine>
```
Comment (cartcrom) [inline src/components/Tokens/TokenTable/TokenRow.tsx:578]: Can we wrap all of this in a separate imported component instead, including all the vars from L487-497

File: src/graphql/data/Token.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ const topTokensQuery = graphql`
     }
   }
 `
-const tokenPricesFragment = graphql`
+export const tokenPricesFragment = graphql`
```
Comment (cartcrom) [inline src/graphql/data/Token.ts:110]: Does this still need to be exported

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 3]
```diff
@@ -543,7 +561,20 @@ export default function LoadedRow({
...
-            <ParentSize>{({ width, height }) => <SparklineChart width={width} height={height} />}</ParentSize>
+            <ParentSize>
+              {({ width, height }) => (
+                <LineChart
+                  data={pricePoints}
+                  getX={(p: PricePoint) => widthScale(p.timestamp)}
+                  getY={(p: PricePoint) => rdScale(p.value)}
+                  curve={curveCardinal.tension(curveTension)}
+                  color={pricePercentChange && pricePercentChange < 0 ? theme.accentFailure : theme.accentSuccess}
+                  strokeWidth={2}
```
Comment (cartcrom) [inline src/components/Tokens/TokenTable/TokenRow.tsx:572]: Can we try 1.5px with here? I think that's how figma is

**Review by cartcrom — CHANGES_REQUESTED** [review]

File: src/graphql/data/Token.ts (hunk 2 above)
Comment (lynnshaoyu) [reply src/graphql/data/Token.ts]: nope ur right :P thx for catch

File: src/components/Tokens/TokenTable/TokenRow.tsx (hunk 3 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenTable/TokenRow.tsx]: done ty

File: src/components/Tokens/TokenTable/TokenRow.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenTable/TokenRow.tsx]: ooh yes moved everything to sparkline charts file

**Review by cartcrom — APPROVED** [review]

**Pull request merged**